sdk-springboot/src/main/java/io/dapr/springboot/DaprController.java

```diff
@@ -45,17 +47,43 @@ public String daprConfig() throws Exception {
         generator.writeString(actorClass);
       }
       generator.writeEndArray();
-      generator.writeStringField("actorIdleTimeout", "10s");
-      generator.writeStringField("actorScanInterval", "1s");
-      generator.writeStringField("drainOngoingCallTimeout", "1s");
-      generator.writeBooleanField("drainBalancedActors", true);
+      // TODO: handle configuration.
       generator.writeEndObject();
       generator.close();
       writer.flush();
       return writer.toString();
     }
   }
 
+  @RequestMapping("/dapr/subscribe")
+  public String daprSubscribe() throws Exception {
+    try (Writer writer = new StringWriter()) {
+      JsonGenerator generator = JSON_FACTORY.createGenerator(writer);
+      generator.writeStartArray();
+      for (String topic : Dapr.getInstance().getSubscribedTopics()) {
+        generator.writeString(topic);
+      }
+      generator.writeEndArray();
+      generator.close();
+      writer.flush();
+      return writer.toString();
+    }
+  }
+
+  @RequestMapping(method = RequestMethod.POST, path = "/{name}")
+  public Mono<byte[]> invokeMethodOrTopic(@PathVariable("name") String name,
+                                          @RequestBody(required = false) byte[] body,
+                                          @RequestHeader Map<String, String> header) {
+    return Dapr.getInstance().handleInvocation(name, body, header);
+  }
+
+  @RequestMapping(method = RequestMethod.PUT, path = "/{name}")
+  public Mono<byte[]> invokeMethodOrTopicViaPut(@PathVariable("name") String name,
+                                          @RequestBody(required = false) byte[] body,
+                                          @RequestHeader Map<String, String> header) {
+    return Dapr.getInstance().handleInvocation(name, body, header);
+  }
+
   @RequestMapping(method = RequestMethod.POST, path = "/actors/{type}/{id}")

```

Comment:
latest version of spring provides a @PostMapping annotation, why not used intead?

Reply:
Thanks. Will do.